source/common/http/conn_manager_impl.cc

```diff
@@ -833,6 +833,13 @@ ConnectionManagerImpl::ActiveStream::ActiveStream(ConnectionManagerImpl& connect
         filter_manager_.log(AccessLog::AccessLogType::DownstreamPeriodic);
         refreshAccessLogFlushTimer();
       }
+      const SystemTime now = connection_manager_.timeSource().systemTime();
+      filter_manager_.streamInfo().getDownstreamBytesMeter()->takeDownstreamPeriodicLoggingSnapshot(
```

Comment:
It take me a bit to convince myself, but getDownstreamBytesMeter is only guaranteed to be non-null because this ActiveStream and the timer event are created on the same thread that sets the meter (in ConnectionManagerImpl::newStream).

Might be worth a comment?

Reply:
Done. Thank you for the review!